playbooks/hc-ansible-deployment/tasks/he_deployment.yml

```diff
@@ -1,23 +1,48 @@
 ---
-- name: Pre Installation Steps
+- name: Install packages and bootstrap local engine VM
   hosts: localhost
   connection: local
   vars:
     he_install_packages: true
     he_pre_checks: true
     he_initial_clean: true
     he_bootstrap_local_vm: true
+    ovirt_repositories_ovirt_release_rpm: "{{ ovirt_repo_release_rpm }}"
```

Comment:
can package installation be done only when the release rpm is provided. For instance, on node - we do not need pkg installation

Reply:
For node which doesn't require pkg to be installed then ovirt_repositories_ovirt_release_rpm is not required.This is optional.

Reply:
So if that variable is not defined, does this playbook need to be edited?